governance/votes/Executive vote - July 11, 2024.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Template - [Executive Vote] Aave-SparkLend Revenue Share, Update Keeper Network Address, Stability Scope Parameter Changes, Delegate Compensation, Spark Proxy Spell - July 11, 2024
+summary: Transfer Dai to Aave as part of the SparkLend revenue sharing agreement, update the Chainlink Keeper Network Treasury Address, make Stability Fee and DSR changes as per the Stability Scope, and trigger a Spark Proxy Spell.
+date: 2024-07-11T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Aave Revenue Share, Update Keeper Network Address, Stability Scope Parameter Changes, Delegate Compensation, Spark Proxy Spell - July 11, 2024
+
+The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- **219,125 DAI** will be transferred to Aave as part of the Aave-SparkLend Revenue Sharing Agreement.
+- The Chainlink Keeper Network Treasury Address will be updated.
+- Multiple Stability Fees and the DSR will be updated.
+- **209.17 MKR** will be distributed to qualified Aligned Delegates.
+- A Spark Proxy Spell will be triggered at [0x91824fa4fd51E8440a122ffDd49C701F5C56D58e](https://etherscan.io/address/0x91824fa4fd51E8440a122ffDd49C701F5C56D58e).
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
```

Comment:
```suggestion
**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
```
Following the previous Executives and the comment above from VoteWizard, we should replace "changes and additions" with "actions".